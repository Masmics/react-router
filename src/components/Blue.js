import React from 'react';

export default function Blue() {
  return (
    <>
      <h1>Blue</h1>
      <div style={{ width: '100px', height: '100px', backgroundColor: '#0000ff' }}></div>
    </>
  );
}
