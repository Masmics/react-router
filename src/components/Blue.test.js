import React from 'react';
import { shallow } from 'enzyme';
import Blue from './Blue';

describe('Blue component', () => {
  it('renders the blue', () => {
    const wrapper = shallow(<Blue />);
    expect(wrapper).toMatchSnapshot();
  });
});
