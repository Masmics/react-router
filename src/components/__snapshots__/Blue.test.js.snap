// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blue component renders the blue 1`] = `
<Fragment>
  <h1>
    Blue
  </h1>
  <div
    style={
      Object {
        "backgroundColor": "#0000ff",
        "height": "100px",
        "width": "100px",
      }
    }
  />
</Fragment>
`;
